examples/auction/src/lib.rs

```diff
@@ -3,206 +3,191 @@ use core::fmt::Debug;
 
 /// # Implementation of an auction smart contract
 ///
-/// To bid, participants send CCD using the bid function.
-/// The participant with the highest bid wins the auction.
-/// Bids are to be placed before the auction end. After that, bids are refused.
-/// Only bids that exceed the highest bid are accepted.
-/// Bids are placed incrementally, i.e., an account's bid is considered
-/// to be the **sum** of all bids.
+/// Users can call the bid function to participate in the auction.
```

Comment:
In our documentation we are not really refering to the contract caller as _user_, often we use address, account or contract, where address is used when it could be either a contract or an account.

In this case the auction only allow bids from accounts, so maybe use account instead?